static/js/main.js

```diff
@@ -32,27 +32,35 @@ $("document").ready(function() {
   
       // an array that will be populated with substring matches
       matches = [];
-  
+
+      if (q === 'R72' || q === 'r72' || q === 'R72 ' || q === 'r72 ') {
```

Comment:
I think this would be better handled with a regex, then you can look for any kind of whitespace.